.github/workflows/test-dev.yml

```
@@ -0,0 +1,48 @@
+name: Test-Dev
+
+on:
+  pull_request:
+    branches: [ "dev" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)